### PR TITLE
Add cd repl function and get/set current lisp directory

### PR DESCRIPTION
**Pull request by Sasanidas**
Also other command from SLIME/Sly

### CODE REVIEW

File: modes/lisp-mode/lisp-mode.lisp [hunk 1]
```diff
@@ -319,6 +319,16 @@
...
 
+(define-command lisp-current-directory () ()
+  (message "Current directory: ~a" *default-pathname-defaults*) )
+
+(define-command lisp-set-directory (&key directory) ()
+  (unless directory
+    (setf directory
+           (prompt-for-directory "New directory: " :directory (buffer-directory))))
+  (swank:set-default-directory 
+   (swank-backend:filename-to-pathname directory)))
```
Comment (cxxxr) [inline modes/lisp-mode/lisp-mode.lisp:330]: Perhaps this only works for self-connection?
If you are connected to another process, you may want to use lisp-eval or something similar.
https://github.com/lem-project/lem/blob/main/modes/lisp-mode/lisp-mode.lisp#LL259C5-L259C5

**Review by cxxxr — COMMENTED** [review]
Thank you for your excellent Pull Request! 
I have left comments.

File: modes/lisp-mode/lisp-mode.lisp (hunk 1 above)
Comment (Sasanidas) [reply modes/lisp-mode/lisp-mode.lisp]: Oh, you are right indeed! 

Thanks for the catch :+1: , I update the code to reflect it

**Comment by cxxxr** [comment]
Thank you!

**Pull request merged**